basic_module/account/src/traits/core.rs

```diff
@@ -0,0 +1,38 @@
+pub mod handles {
+    pub use crate::account::AccountError;
+    pub use crate::Transaction;
+    pub use coordinator::context::SubStorageAccess;
+    pub use key::{Address, Error as KeyError, Password, Private, Public, Signature};
+
+    pub trait CheckTxHandler {
+        fn verify_transaction(tx: &Transaction) -> bool;
+    }
+
+    pub trait TransactionExecuter {
+        fn execute_transactions(context: &mut dyn SubStorageAccess, transactions: &[Transaction]);
+    }
+
+    pub trait AccountReviser {
+        fn add_balance(address: &Address, val: u64) -> Result<(), AccountError>;
+
+        fn sub_balance(address: &Address, val: u64) -> Result<(), AccountError>;
+
+        fn set_balance(address: &Address, val: u64) -> Result<(), AccountError>;
+
+        fn increase_sequence(address: &Address) -> Result<(), AccountError>;
+    }
+
+    pub trait AccountView {
+        fn has_account(address: &Address) -> bool;
+
+        fn is_active(address: &Address) -> Result<bool, AccountError>;
+
+        fn get_balance(context: &mut dyn SubStorageAccess, address: &Address) -> Option<u64>;
```

Comment:
The interface should not expect the context as an input, account data is managed by `Account` module so that must be resolved internally.